### ISSUE TITLE
Details/summary in documentation body isn't behaving as expected with tabbing & opening 

### ISSUE THREAD

**Issue by ststimac** [webhintio/webhint.io#453]
This feature we have in the docs content has two issues: 
![not-accessible](https://user-images.githubusercontent.com/18073131/41067306-ec74fb06-6999-11e8-87e7-622c35b5bb56.PNG)

1. The cursor when hovered over it is a text cursor, not a pointer. Needs to be a pointer for the visual cue that this is something that can be clicked on.

2. It currently can't be tabbed into via the keyboard and won't open when enter is pressed so something is happening there. There's no focus state being triggered by tabbing either. Something isn't implemented correctly so we'll have to check the code. The behavior should be similar to what happens on the status page of EdgePortal in terms of tabbing into details/summary and hitting enter.

**Comment by molant** [comment]
Except Edge all the other browsers support details/summary so we shouldn't have to do anything to make it accessible 🤔

**Comment by ststimac** [comment]
Something is happening on our page then with the implementation because you can't tab into it and open it via the keyboard right now.

**Comment by ststimac** [comment]
That's what I mean by accessible, something isn't working correctly and I don't know what yet. For reference, I checked tabbing on edgeportal's status page and hitting enter to expand the details and it worked fine. Doesn't work on our page, and I'm in Edge for both of those. Will have to do some debugging.

**Comment by Malvoz** [comment]
@ststimac 
The MS platform status page has a &lt;details&gt;/&lt;summary&gt; polyfill implemented, that is why the tabbing works when you interact with the element using Edge. 

This is because what @molant said, the feature is still a work in progress for Edge: https://developer.microsoft.com/en-us/microsoft-edge/platform/status/detailssummary/

What Edge does when:

**Details is open**
```
<details role="group" open>
<summary tabindex="0" role="button" aria-expanded="true"></summary>
</details>
```
// Actually they do `open="open"`, but this is a boolean attr so should just be `open`.

**Details is closed**
```
<details role="group">
<summary tabindex="0" role="button" aria-expanded="false"></summary>
</details>
```
Also about the cursor, idk why user agents haven't yet decided to set  `cursor: pointer` for `<summary>` by default, may be related to this: https://github.com/w3c/csswg-drafts/issues/1936

**Comment by molant** [comment]
@sarvaje is taking care of this with the new results page.

**Comment by molant** [comment]
@sarvaje I think this is fixed, right?

**Comment by sarvaje** [comment]
2 is already done.

1 is still happening, and it happens in all browsers.

I will work on this today.
